src/main/java/meteordevelopment/meteorclient/gui/WidgetScreen.java

```diff
@@ -70,9 +72,7 @@ public WidgetScreen(GuiTheme theme, String title) {
         if (parent != null) {
             animProgress = 1;
 
-            if (this instanceof TabScreen && parent instanceof TabScreen) {
-                parent = ((TabScreen) parent).parent;
-            }
+            if (this instanceof TabScreen && parent instanceof TabScreen) parent = ((TabScreen) parent).parent;
```

Comment:
👎

Reply:
imo this one is fine if you use pattern matching to remove the cast

Reply:
e.g. `if (this instanceof TabScreen && parent instanceof TabScreen parentTabScreen) parent = parentTabScreen.parent;`